test/torch_np/numpy_tests/core/test_shape_base.py

```diff
@@ -26,13 +26,18 @@
         atleast_1d,
         atleast_2d,
         atleast_3d,
-        AxisError,
         concatenate,
         hstack,
         newaxis,
         stack,
         vstack,
     )
+
+    if int(numpy.__version__[0]) >= 2:
+        from numpy.exceptions import AxisError
+    else:
+        from numpy import AxisError
```

Comment:
I thought PT is already compatible with numpy 2, or am I imagining things?

Reply:
The support might not be fully completed. Our test coverage also wasn't there since we didn't test numpy 2.0+ until 3.13.